### PR TITLE
v0.3.2: Fix bug with payable proxy

### PR DESCRIPTION
**Pull request by BrendanChou**
Client was sending funds to payable proxy even when withdrawing or withdrawing to zero

### CODE REVIEW

File: dydx/client.py [hunk 1]
```diff
@@ -146,38 +146,45 @@ def _operate(
...
-            'otherAddress':
-                consts.PAYABLE_PROXY_ADDRESS if market == 0 else otherAddress,
+            'otherAddress': actualWithdrawOrDepositAddress,
             'otherAccountId': 0,
             'data': '0x'
         }]
+        txOptions = dict(
+            value=(
+                wei
+                if (isDeposit and market == consts.MARKET_WETH)
```
Comment (AntonioJuliano) [inline dydx/client.py]: One thing I had on the typescript client was both an ETH and a WETH market, and would only use the proxy for the ETH one, which could be valuable for market makers that already have WETH

File: dydx/client.py [hunk 2]
```diff
@@ -146,38 +146,45 @@ def _operate(
             'ref': ref,
             'value': wei
         }
+        actualWithdrawOrDepositAddress = (
```
Comment (AntonioJuliano) [inline dydx/client.py]: Does this actually change anything? Isn’t MARKET_WETH = 0?

File: dydx/client.py [hunk 3]
```diff
@@ -146,38 +146,45 @@ def _operate(
...
 
-        if (market == 0):
+        if (market == consts.MARKET_WETH):
             return self._send_eth_transaction(
                 self.payable_proxy.functions.operate(
                     accounts,
                     operations,
                     otherAddress
                 ),
-                options=dict(
```
Comment (AntonioJuliano) [inline dydx/client.py]: Was this the bug?

Comment (BrendanChou) [reply dydx/client.py]: ya

File: dydx/client.py (hunk 2 above)
Comment (BrendanChou) [reply dydx/client.py]: doesn't change any logic, no

File: dydx/client.py (hunk 1 above)
Comment (BrendanChou) [reply dydx/client.py]: Sure, can add that in a later change

**Pull request merged**